locations/spiders/synlab_fr.py

```
@@ -0,0 +1,58 @@
+    def extract_json(self, response):
+        return chompjs.parse_js_object(
+            response.xpath('//script[contains(text(), "const villes = ")]/text()').get().split("const villes =")[1]
+        )
+
+    def post_process_item(self, item, response, location):
+        apply_category(Categories.MEDICAL_LABORATORY, item)
+        name = location.pop("nom","").lower()
+
+        if name.startswith("synlab"):
+            item["branch"] = (name
+                .removeprefix("synlab ")
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)